pgclone/storage.py

```diff
@@ -18,32 +22,39 @@ def validate_s3_support():  # pragma: no cover
         )
 
 
-class Storage:
-    def __init__(self, storage_location):
+class _Storage(abc.ABC):
```

Comment:
We're basically using this as an `ABC` already it allows us to avoid ` | None` on base methods.